src/patternfly/components/Card/card-image.hbs

```diff
@@ -1,6 +1,6 @@
-<div class="pf-c-card__image{{#if card-image--modifier}} {{card-image--modifier}}{{/if}}"
-  {{#if card-image--attribute}}
-    {{{card-image--attribute}}}
+<div class="pf-c-card__main{{#if card-main--modifier}} {{card-main--modifier}}{{/if}}"
```

Comment:
can we call it `__head-main` instead, since it should only exist in `__head`? I think we should reserve `__main` in case we want to add that to the root of the card.

Reply:
yeah +1